.github/workflows/pr-conflict-checker.yml

```diff
@@ -1,14 +1,7 @@
 name: Prowler - PR Conflict Checker
 
 on:
-  pull_request:
```

Comment:
I removed this because this needs to work for forks too. So it needs to run when `pull_request_target` to run in the Prowler's repo context.